sys/dev/ath/if_ath_usb_devlist.h

```diff
@@ -0,0 +1,81 @@
+/*-
+ * Copyright (c) 2011 Damien Bergamini <damien.bergamini@free.fr>
+ * Copyright (c) 2018 Stefan Sperling <stsp@openbsd.org>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+/*
+ * USB front-end for Atheros AR9271 and AR7010 chipsets.
+ */
+
+#include <dev/usb/usbdi.h>
+
+/*	$OpenBSD: usbdevs.h,v 1.768 2023/05/10 18:26:43 miod Exp $	*/
+#define	USB_VENDOR_ATHEROS2					0x0cf3		/* Atheros Communications */
+#define	USB_VENDOR_AZUREWAVE				0x13d3		/* AsureWave */
+#define	USB_VENDOR_DLINK2					0x07d1		/* D-Link */
+#define	USB_VENDOR_LITEON					0x04ca		/* Lite-On Technology */
+#define	USB_VENDOR_NETGEAR					0x0846		/* BayNETGEAR */
+#define	USB_VENDOR_VIA						0x040d		/* VIA */
+#define	USB_VENDOR_ACCTON					0x083a		/* Accton Technology */
+#define	USB_VENDOR_ACTIONTEC				0x1668		/* Actiontec Electronics */
+#define	USB_VENDOR_PANASONIC				0x04da		/* Panasonic (Matsushita) */
+#define	USB_VENDOR_MELCO					0x0411		/* Melco */
+
+#define	USB_PRODUCT_ATHEROS2_AR9271_1		0x1006		/* AR9271 */
```

Comment:
Could we align all columns?

Reply:
All columns were align with tab, in vscode looks good. I will use spaces instead.